client/app/src/Controller/Ndr/ActionController.php

```diff
@@ -36,39 +37,50 @@ class ActionController extends AbstractController
         'ndr-action-more-info',
     ];
 
+    /**
+     * @var ClientApi
+     */
+    private $clientApi;
+
     public function __construct(
         ReportApi $reportApi,
         RestClient $restClient,
-        StepRedirector $stepRedirector
+        StepRedirector $stepRedirector,
+        ClientApi $clientApi
```

Comment:
It doesn't really matter for this, but if you do come across/need to add to the constructor, refactor it & use [constructor promotion](https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion)